synapse/cores/common.py

```diff
@@ -2347,20 +2347,6 @@ def _initPropTufo(self, tufo):
         self.addTufoProp(form, prop, **info)
         return
 
-    def _tufosByIn(self, prop, valus, limit=None):
```

Comment:
This function is defined twice with the same content - see https://github.com/vertexproject/synapse/blob/master/synapse/cores/common.py#L2273 and https://github.com/vertexproject/synapse/blob/master/synapse/cores/common.py#L2350